app/src/main/java/Fragments/ItemDescriptionFragment.java

```diff
@@ -113,7 +114,18 @@ public void setDisplayViews(){
             description_tv.setTypeface(null);
             description_tv.setText(this.itemToDisplay.getDescription());
         }
-        new DownloadImageTask(photo_iv).execute(itemToDisplay.getImageURL());
+
+        String imgUrl = itemToDisplay.getImageURL();
```

Comment:
Instead of repeating the same code three times, you should add this as a method in the helper class that we have!

Reply:
Will do!